tensorflow/lite/tools/cmake/modules/Findgemmlowp.cmake

```diff
@@ -19,11 +19,7 @@ include(gemmlowp)
 if(gemmlowp_POPULATED)
   set(GEMMLOWP_FOUND TRUE)
   get_target_property(GEMMLOWP_INCLUDE_DIRS gemmlowp INTERFACE_DIRECTORIES)
-  set(GEMMLOWP_LIBRARIES
-    gemmlowp
-    gemmlowp_fixedpoint
-    gemmlowp_profiler
-    gemmlowp_eight_bit_int_gemm
-  )
+  add_library(gemmlowp::gemmlowp ALIAS gemmlowp)
```

Comment:
It looks like this library has a conflict with another dependency ([google_benchmark](github.com/google/benchmark/blob/main/src/CMakeLists.txt#L20)).
```
CMake Error at .../tflite_build/google_benchmark/src/CMakeLists.txt:20 (add_library):
  add_library cannot create target "benchmark" because another target with
  the same name already exists.  The existing target is an executable created
  in source directory ".../tflite_build/gemmlowp/contrib".
  See documentation for policy CMP0002 for more details.
```

Reply:
Unfortunately gemmlowp creates a benchmark target if `BUILD_TESTING` is set (see [CMakeLists.txt](https://github.com/google/gemmlowp/blob/master/contrib/CMakeLists.txt#L83)).
The only good option that I can come up with is saving the value of `BUILD_TESTING`, setting it to OFF, calling `add_subdirectory` for gemmlowp and restoring the value afterwards:

```cmake
set(BUILD_TESTING_TMP ${BUILD_TESTING})
set(BUILD_TESTING OFF)

add_subdirectory(
  "${gemmlowp_SOURCE_DIR}/contrib"
  "${gemmlowp_BINARY_DIR}"
)

set(BUILD_TESTING ${BUILD_TESTING_TMP})
```

Reply:
I see, I think this should be fine. Let me check internally if this would be an acceptable solution. Can you add a comment before setting `BUILD_TESTING_TMP` that explains why this is needed?

Something along:

> Unfortunately gemmlowp creates a benchmark target if BUILD_TESTING is set (see [CMakeLists.txt](https://github.com/google/gemmlowp/blob/master/contrib/CMakeLists.txt#L83)) that clashes with a target created by google_benchmark.

Reply:
I'm approving to get the CI to run on the change to see if anything else is blocking.